test/controllers/actions/AuthActionSpec.scala

```diff
@@ -427,11 +439,21 @@ object AuthActionSpec {
     def onPageLoad(mode:Mode=NormalMode) = (authAction andThen allowAccess(mode)) { _ => Ok }
   }
 
-  private def fakeMinimalPsaConnector(isSuspended:Boolean = false) : MinimalPsaConnector = {
-    new MinimalPsaConnector {
-      override def isPsaSuspended(psaId: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Boolean] = {
-        Future.successful(isSuspended)
-      }
+  class FakeMinimalPsaConnector(isSuspended:Boolean) extends MinimalPsaConnector {
+
+    def verifyIsCalled():Boolean = isCalled
+
+    private var isCalled: Boolean = false
```

Comment:
Could we use Mockito to verifies instead of var ?

Reply:
Done